packages/babel-plugin-proposal-unicode-property-regex/package.json

```diff
@@ -8,9 +8,6 @@
     "access": "public"
   },
   "main": "lib/index.js",
-  "engines": {
```

Comment:
The `engines` field is redundant here as long as it peer depends on `@babel/core`.